WSL brew cask error

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

> brew upgrade oh-my-posh
Error: Failed to load cask: oh-my-posh
Cask 'oh-my-posh' definition is invalid: invalid 'sha256' value: nil
Warning: Treating oh-my-posh as a formula.
Warning: jandedobbeleer/oh-my-posh/oh-my-posh 29.15.1 already installed
Warning: jandedobbeleer/oh-my-posh/oh-my-posh 29.15.1 already installed

### Theme

customized agnoster

### What OS are you seeing the problem on?

Linux

### Which shell are you using?

bash

### Log output

```Shell
> oh-my-posh debug --plain

Version: 29.15.1

Shell: shell (5.3.9(1)-release)

Prompt:

╭─ WSL at    ~                                                                                         ✔  10:55:23  
╰─

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Path(true)                                 -   0 ms
Git(false)                                 -   0 ms
Node(false)                                -   0 ms
Go(false)                                  -   0 ms
Julia(false)                               -   0 ms
Python(false)                              -   1 ms
Ruby(false)                                -   0 ms
Azfunc(false)                              -   0 ms
Aws(false)                                 -   0 ms
Root(false)                                -   0 ms
Executiontime(false)                       -   0 ms
Exit(true)                                 -   1 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms

Run duration: 10.322912ms

Cache path: /home/anwer/.cache/oh-my-posh

Config path: /mnt/c/users/Najeeb Anwer/.mytheme.omp.json

Logs:

[DEBUG] 10:55:23.434 debug.go:39 → logging enabled, raw mode: true
[DEBUG] 10:55:23.434 store.go:]:163 → (session) key not found: prompt_count_cache
[TRACE] 10:55:23.434 store.go:](session prompt_count_cache) - 22.918µs
[TRACE] 10:55:23.434 terminal.go:setPromptCount() - 28.407µs
[TRACE] 10:55:23.434 terminal.go:GOOS() - 46ns
[DEBUG] 10:55:23.434 terminal.go:setPwd:102 → /home/anwer
[TRACE] 10:55:23.434 terminal.go:setPwd() - 20.68µs
[TRACE] 10:55:23.434 terminal.go:Init() - 59µs
[DEBUG] 10:55:23.434 home.go:12 → /home/anwer
[TRACE] 10:55:23.434 path.go:Path() - 13.515µs
[DEBUG] 10:55:23.435 store.go:init:95 → (device) loading INITVERSIONBASH
[DEBUG] 10:55:23.435 store.go:init:95 → (device) loading environment_platform
[DEBUG] 10:55:23.435 store.go:init:95 → (device) loading DSC_CONFIGURATION
[DEBUG] 10:55:23.435 store.go:init:95 → (device) loading DSC_SHELL
[DEBUG] 10:55:23.435 store.go:init:95 → (device) loading is_wsl
[TRACE] 10:55:23.435 store.go:init(device omp.cache) - 826.776µs
[TRACE] 10:55:23.435 init.go:SessionID() - 1.178µs
[TRACE] 10:55:23.435 path.go:Path() - 59ns
[DEBUG] 10:55:23.435 store.go:init:95 → (session) loading CONFIG
[DEBUG] 10:55:23.435 store.go:init:95 → (session) loading CONFIG_SOURCE
[DEBUG] 10:55:23.435 store.go:init:95 → (session) loading prompt_count_cache
[DEBUG] 10:55:23.435 store.go:init:95 → (session) loading toggle_cache
[TRACE] 10:55:23.435 store.go:init(session bash.6afbc191-1477-4c1c-b2ce-571868cc067c.omp.cache) - 436.51µs
[DEBUG] 10:55:23.435 store.go:]:163 → (device) key not found: reload
[TRACE] 10:55:23.435 store.go:](device reload) - 7.001µs
[DEBUG] 10:55:23.435 store.go:]:176 → (session) found entry: CONFIG - /gIHfwMBAQZDb25maWcB/4AAASABB1BhbGV0dGUB/4IAAQtEZWJ1Z1Byb21wdAH/hAABA1ZhcgH/jgABCFBhbGV0dGVzAf+QAAEJVmFsaWRMaW5lAf+EAAEPU2Vjb25kYXJ5UHJvbXB0Af+EAAEPVHJhbnNpZW50UHJvbXB0Af+EAAEJRXJyb3JMaW5lAf+EAAEETWFwcwH/lAABB1VwZ3JhZGUB/5gAAQdFeHRlbmRzAQwAAQtBY2NlbnRDb2xvcgEMAAEUQ29uc29sZVRpdGxlVGVtcGxhdGUBDAABA1BXRAEMAAEGU291cmNlAQwAAQZGb3JtYXQBDAABElRlcm1pbmFsQmFja2dyb3VuZAEMAAEOVG9vbFRpcHNBY3Rpb24BDAABBkJsb2NrcwH/ngABBUN5Y2xlAf+iAAENSVRlcm1GZWF0dXJlcwH/pAABCFRvb2x0aXBzAf+cAAEHVmVyc2lvbgEEAAENTWlncmF0ZUdseXBocwECAAEFQXN5bmMBAgABEFNoZWxsSW50ZWdyYXRpb24BAgABCkZpbmFsU3BhY2UBAgABDVVwZ3JhZGVOb3RpY2UBAgABDlBhdGNoUHdzaEJsZWVkAQIAAQtBdXRvVXBncmFkZQECAAEXRW5hYmxlQ3Vyc29yUG9zaXRpb25pbmcBAgABCVN0cmVhbWluZwEEAAAAF/+BBAEBB1BhbGV0dGUB/4IAAQwBDAAA/gIx/4MDAQEHU2VnbWVudAH/hAABJAEHT3B0aW9ucwH/hgABClByb3BlcnRpZXMB/4YAAQVDYWNoZQH/iAABBUFsaWFzAQwAAQ5MZWFkaW5nRGlhbW9uZAEMAAEPVHJhaWxpbmdEaWFtb25kAQwAAQhUZW1wbGF0ZQEMAAEKRm9yZWdyb3VuZAEMAAEOVGVtcGxhdGVzTG9naWMBDAABD1Bvd2VybGluZVN5bWJvbAEMAAEKQmFja2dyb3VuZAEMAAEGRmlsbGVyAQwAAQRUeXBlAQwAAQVTdHlsZQEMAAEWTGVhZGluZ1Bvd2VybGluZVN5bWJvbAEMAAELUGxhY2Vob2xkZXIBDAABBFRpcHMB/4oAARNCYWNrZ3JvdW5kVGVtcGxhdGVzAf+MAAEJVGVtcGxhdGVzAf+MAAEORXhjbHVkZUZvbGRlcnMB/4oAAQ5JbmNsdWRlRm9sZGVycwH/igABBU5lZWRzAf+KAAETRm9yZWdyb3VuZFRlbXBsYXRlcwH/jAABBUluZGV4AQQAAQhNaW5XaWR0aAEEAAEIRHVyYXRpb24BBAABCk5hbWVMZW5ndGgBBAABCE1heFdpZHRoAQQAAQdUaW1lb3V0AQQAAQdOZXdsaW5lAQIAAQdFbmFibGVkAQIAAQ9JbnZlcnRQb3dlcmxpbmUBAgABBUZvcmNlAQIAAQdUb2dnbGVkAQIAAQdQZW5kaW5nAQIAAQtJbnRlcmFjdGl2ZQECAAAAE/+FBAEBA01hcAH/hgABDAEQAAAt/4cDAQEFQ2FjaGUB/4gAAQIBCER1cmF0aW9uAQwAAQhTdHJhdGVneQEMAAAAFv+JAgEBCFtdc3RyaW5nAf+KAAEMAAAS/4sCAQEETGlzdAH/jAABDAAAJ/+NBAEBF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9Af+OAAEMARAAAC3/jwMBAQhQYWxldHRlcwH/kAABAgEETGlzdAH/kgABCFRlbXBsYXRlAQwAAAAp/5EEAQEYbWFwW3N0cmluZ11jb2xvci5QYWxldHRlAf+SAAEMAf+CAAA//5MDAQEGQ29uZmlnAf+UAAEDAQhVc2VyTmFtZQH/lgABCEhvc3ROYW1lAf+WAAEJU2hlbGxOYW1lAf+WAAAAE/+VBAEBA01hcAH/lgABDAEMAABc/5cDAQEGQ29uZmlnAf+YAAEGAQZTb3VyY2UBDAABCEludGVydmFsAQwAAQZMYXRlc3QBDAABBEF1dG8BAgABDURpc3BsYXlOb3RpY2UBAgABBUZvcmNlAQIAAAAe/50CAQEPW10qY29uZmlnLkJsb2NrAf+eAAH/mgAA/5D/mQMBAv+aAAEKAQRUeXBlAQwAAQlBbGlnbm1lbnQBDAABBkZpbGxlcgEMAAEIT3ZlcmZsb3cBDAABDkxlYWRpbmdEaWFtb25kAQwAAQ9UcmFpbGluZ0RpYW1vbmQBDAABCFNlZ21lbnRzAf+cAAEHTmV3bGluZQECAAEFRm9yY2UBAgABBUluZGV4AQQAAAAg/5sCAQERW10qY29uZmlnLlNlZ21lbnQB/5wAAf+EAAAU/6ECAQEFQ3ljbGUB/6IAAf+gAAAq/58DAQL/oAABAgEKQmFja2dyb3VuZAEMAAEKRm9yZWdyb3VuZAEMAAAAG/+jAgEBDUlUZXJtRmVhdHVyZXMB/6QAAQwAAP4KCP+ACgEDY2RuAQQxNjhoAAMde3sgLlNoZWxsIH19IGluIHt7IC5Gb2xkZXIgfX0CKy9tbnQvYy91c2Vycy9OYWplZWIgQW53ZXIvLm15dGhlbWUub21wLmpzb24BBGpzb24DAwEGcHJvbXB0AQRsZWZ0BQMFCeKVreKUgO6CtgIoIHt7IGlmIC5XU0wgfX1XU0wgYXQge3sgZW5kIH19e3suSWNvbn19IAEHIzAwMDAwMAMHI2QzZDdjZgICb3MBB2RpYW1vbmQAAQIFc3R5bGUGc3RyaW5nDAYABGZ1bGwJaG9tZV9pY29uBnN0cmluZwwDAAF+BhEg74G8IHt7IC5QYXRoIH19IAEHI2U0ZTRlNAID7oKwAQcjMzQ2NWE0AgRwYXRoAQlwb3dlcmxpbmUAAQMLYnJhbmNoX2ljb24Gc3RyaW5nDAYABO+EpiARZmV0Y2hfc3Rhc2hfY291bnQEYm9vbAICAAETZmV0Y2hfdXBzdHJlYW1faWNvbgRib29sAgIAAQb/9CB7eyAuSEVBRCB9fXt7aWYgLkJyYW5jaFN0YXR1cyB9fSB7eyAuQnJhbmNoU3RhdHVzIH19e3sgZW5kIH19e3sgaWYgLldvcmtpbmcuQ2hhbmdlZCB9fSDvgYQge3sgLldvcmtpbmcuU3RyaW5nIH19e3sgZW5kIH19e3sgaWYgYW5kICguU3RhZ2luZy5DaGFuZ2VkKSAoLldvcmtpbmcuQ2hhbmdlZCkgfX0gfHt7IGVuZCB9fXt7IGlmIC5TdGFnaW5nLkNoYW5nZWQgfX0g74GGIHt7IC5TdGFnaW5nLlN0cmluZyB9fXt7IGVuZCB9fSABByMwMDAwMDACA+6CsAEHIzRlOWEwNgIDZ2l0AQlwb3dlcmxpbmUEBEF7eyBpZiBvciAoLldvcmtpbmcuQ2hhbmdlZCkgKC5TdGFnaW5nLkNoYW5nZWQpIH19I2M0YTAwMHt7IGVuZCB9fTl7eyBpZiBhbmQgKGd0IC5BaGVhZCAwKSAoZ3QgLkJlaGluZCAwKSB9fSNmMjZkNTB7eyBlbmQgfX0ke3sgaWYgZ3QgLkFoZWFkIDAgfX0jNGU5YTA2e3sgZW5kIH19JXt7IGlmIGd0IC5CZWhpbmQgMCB9fSM4OWQxZGN7eyBlbmQgfX0AAAEGcHJvbXB0AQVyaWdodAULAQENZmV0Y2hfdmVyc2lvbgRib29sAgIAAQZQIHt7IGlmIC5QYWNrYWdlTWFuYWdlckljb24gfX17eyAuUGFja2FnZU1hbmFnZXJJY29uIH19IHt7IGVuZCB9fXt7IC5GdWxsIH19IO+imCABByNmZmZmZmYCA+6CsgEHIzY4OWY2MwIEbm9kZQEJcG93ZXJsaW5lEgEAAQENZmV0Y2hfdmVyc2lvbgRib29sAgIAAQY/IHt7IGlmIC5FcnJvciB9fXt7IC5FcnJvciB9fXt7IGVsc2UgfX17eyAuRnVsbCB9fXt7IGVuZCB9fSDumKcgAQcjMTExMTExAgPugrIBByMwMGFjZDcCAmdvAQlwb3dlcmxpbmUSAQABAQ1mZXRjaF92ZXJzaW9uBGJvb2wCAgABBj8ge3sgaWYgLkVycm9yIH19e3sgLkVycm9yIH19e3sgZWxzZSB9fXt7IC5GdWxsIH19e3sgZW5kIH19IO6YpCABByMxMTExMTECA+6CsgEHIzQwNjNEOAIFanVsaWEBCXBvd2VybGluZRIBAAEDEWZldGNoX3ZpcnR1YWxfZW52BGJvb2wCAgAADGRpc3BsYXlfbW9kZQZzdHJpbmcMBwAFZmlsZXMNZmV0Y2hfdmVyc2lvbgRib29sAgIAAQZiIHt7IGlmIC5FcnJvciB9fXt7IC5FcnJvciB9fXt7IGVsc2UgfX17eyBpZiAuVmVudiB9fXt7IC5WZW52IH19IHt7IGVuZCB9fXt7IC5GdWxsIH19e3sgZW5kIH19IO6ItSABByMxMTExMTECA+6CsgEHI0ZGREU1NwIGcHl0aG9uAQlwb3dlcmxpbmUSAQABAgxkaXNwbGF5X21vZGUGc3RyaW5nDAcABWZpbGVzDWZldGNoX3ZlcnNpb24EYm9vbAICAAEGPyB7eyBpZiAuRXJyb3IgfX17eyAuRXJyb3IgfX17eyBlbHNlIH19e3sgLkZ1bGwgfX17eyBlbmQgfX0g7p6RIAEHI2ZmZmZmZgID7oKyAQcjQUUxNDAxAgRydWJ5AQlwb3dlcmxpbmUSAQABAg1mZXRjaF92ZXJzaW9uBGJvb2wCAgAADGRpc3BsYXlfbW9kZQZzdHJpbmcMBwAFZmlsZXMGPiB7eyBpZiAuRXJyb3IgfX17eyAuRXJyb3IgfX17eyBlbHNlIH19e3sgLkZ1bGwgfX17eyBlbmQgfX0g74OnAQcjZmZmZmZmAgPugrIBByNGRUFDMTkCBmF6ZnVuYwEJcG93ZXJsaW5lEgEAAQEPZGlzcGxheV9kZWZhdWx0BGJvb2wCAgAABjsge3sgLlByb2ZpbGUgfX17eyBpZiAuUmVnaW9uIH19QHt7IC5SZWdpb24gfX17eyBlbmQgfX0g7p6tIAEHI2ZmZmZmZgID7oKyAwNhd3MBCXBvd2VybGluZQQCMHt7aWYgY29udGFpbnMgImRlZmF1bHQiIC5Qcm9maWxlfX0jRkZBNDAwe3tlbmR9fSx7e2lmIGNvbnRhaW5zICJqYW4iIC5Qcm9maWxlfX0jZjExODRje3tlbmR9fQ4BAAEBCXJvb3RfaWNvbgZzdHJpbmcMBQAD74KtBgUg74OnIAEHIzExMTExMQID7oKyAQcjZmZmZjY2AgRyb290AQlwb3dlcmxpbmUSAQAHGCB7eyAuRm9ybWF0dGVkTXMgfX0g74mSIAEHIzAwMDAwMAID7oKyAQcjYzRhMDAwAg1leGVjdXRpb250aW1lAQlwb3dlcmxpbmUSAQABAg5hbHdheXNfZW5hYmxlZARib29sAgIAARFkaXNwbGF5X2V4aXRfY29kZQRib29sAgIAAQY9IHt7IGlmIGd0IC5Db2RlIDAgfX3vgI0ge3sgLk1lYW5pbmcgfX17eyBlbHNlIH194pyUe3sgZW5kIH19IAEHI2QzZDdjZgID7oKyAQcjMDAwMDAwAgRleGl0AQlwb3dlcmxpbmUEASN7eyBpZiBndCAuQ29kZSAwIH19I2NjMjIyMnt7IGVuZCB9fQ4BAAYD7oK0AScge3sgLkN1cnJlbnREYXRlIHwgZGF0ZSAuRm9ybWF0IH19IO+AlyABByMwMDAwMDADByNkM2Q3Y2YCBHRpbWUBB2RpYW1vbmQSAQAAAQZwcm9tcHQBBGxlZnQFAQcG4pWw4pSAAQcjZDNkN2NmBQR0ZXh0AQVwbGFpbgABAQAECAQBAA==
[TRACE] 10:55:23.435 store.go:](session CONFIG) - 36.436µs
[TRACE] 10:55:23.436 gob.go:Restore() - 419.155µs
[TRACE] 10:55:23.436 gob.go:Get() - 467.613µs
[TRACE] 10:55:23.436 terminal.go:Shell() - 116ns
[TRACE] 10:55:23.436 terminal.go:Flags() - 40ns
[DEBUG] 10:55:23.436 store.go:]:163 → (session) key not found: template_cache
[TRACE] 10:55:23.436 store.go:](session template_cache) - 6.157µs
[TRACE] 10:55:23.436 cache.go:restoreCache() - 10.211µs
[TRACE] 10:55:23.436 terminal_unix.go:Root() - 1.061µs
[TRACE] 10:55:23.436 terminal.go:Shell() - 88ns
[TRACE] 10:55:23.436 terminal.go:Flags() - 24ns
[TRACE] 10:55:23.436 terminal.go:StatusCodes() - 56ns
[DEBUG] 10:55:23.436 store.go:]:176 → (device) found entry: is_wsl - true
[TRACE] 10:55:23.436 store.go:](device is_wsl) - 4.639µs
[TRACE] 10:55:23.436 terminal_unix.go:IsWsl() - 37.472µs
[TRACE] 10:55:23.436 terminal.go:Flags() - 51ns
[TRACE] 10:55:23.436 terminal.go:Flags() - 41ns
[DEBUG] 10:55:23.436 home.go:12 → /home/anwer
[DEBUG] 10:55:23.436 store.go:]:176 → (device) found entry: is_wsl - true
[TRACE] 10:55:23.436 store.go:](device is_wsl) - 4.107µs
[TRACE] 10:55:23.436 terminal_unix.go:IsWsl() - 7.301µs
[DEBUG] 10:55:23.437 terminal.go:RunCommand:280 → //wsl.localhost/Ubuntu-Preview/home/anwer
[TRACE] 10:55:23.437 terminal.go:RunCommand(wslpath -m /home/anwer) - 1.171441ms
[TRACE] 10:55:23.437 terminal.go:Flags() - 111ns
[TRACE] 10:55:23.437 terminal.go:Flags() - 25ns
[TRACE] 10:55:23.437 terminal.go:GOOS() - 95ns
[DEBUG] 10:55:23.437 terminal.go:User:236 → anwer
[TRACE] 10:55:23.437 terminal.go:User() - 4.049µs
[DEBUG] 10:55:23.437 terminal.go:Host:253 → DELL-XPS-13
[TRACE] 10:55:23.437 terminal.go:Host() - 11.677µs
[TRACE] 10:55:23.437 terminal.go:GOOS() - 40ns
[DEBUG] 10:55:23.437 store.go:]:176 → (device) found entry: environment_platform - ubuntu
[TRACE] 10:55:23.437 store.go:](device environment_platform) - 41.433µs
[DEBUG] 10:55:23.437 terminal.go:Getenv:64 → 1
[TRACE] 10:55:23.437 terminal.go:Getenv(SHLVL) - 3.959µs
[DEBUG] 10:55:23.437 writer.go:Init:107 → terminal program: Windows Terminal
[DEBUG] 10:55:23.437 writer.go:Init:108 → terminal shell: shell
[DEBUG] 10:55:23.437 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.437 terminal.go:Getenv(OMP_CACHE_DISABLED) - 7.381µs
[DEBUG] 10:55:23.437 store.go:]:163 → (device) key not found: accent_color
[TRACE] 10:55:23.437 store.go:](device accent_color) - 10.652µs
[TRACE] 10:55:23.437 colors.go:SetAccentColor() - 14.448µs
[TRACE] 10:55:23.437 terminal.go:Shell() - 90ns
[DEBUG] 10:55:23.437 terminal.go:Getenv:64 → 5.3.9(1)-release
[TRACE] 10:55:23.437 terminal.go:Getenv(POSH_SHELL_VERSION) - 2.489µs
[DEBUG] 10:55:23.437 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 10:55:23.437 text.go:patchTemplate:149 → {{ .Shell }} in {{ .Folder }}
[TRACE] 10:55:23.438 terminal.go:Flags() - 264ns
[DEBUG] 10:55:23.438 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.438 terminal.go:Getenv(POSH_CURSOR_LINE) - 5.943µs
[DEBUG] 10:55:23.438 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.438 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 2.342µs
[TRACE] 10:55:23.438 terminal.go:Flags() - 44ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 31ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 31ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 109ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 33ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 40ns
[DEBUG] 10:55:23.438 home.go:12 → /home/anwer
[TRACE] 10:55:23.438 terminal.go:GOOS() - 83ns
[DEBUG] 10:55:23.438 segment.go:Execute:173 → segment: Git
[DEBUG] 10:55:23.438 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.438 store.go:](session toggle_cache) - 10.361µs
[DEBUG] 10:55:23.438 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.438 map.go:Bool:133 → disable_with_jj: false
[TRACE] 10:55:23.438 terminal.go:Flags() - 281ns
[DEBUG] 10:55:23.438 terminal.go:ResolveSymlink:195 → /home/anwer
[DEBUG] 10:55:23.438 home.go:12 → /home/anwer
[TRACE] 10:55:23.438 terminal.go:ResolveSymlink(/home/anwer) - 12.196µs
[TRACE] 10:55:23.438 terminal.go:GOOS() - 55ns
[DEBUG] 10:55:23.438 segment.go:Execute:173 → segment: Os
[ERROR] 10:55:23.438 terminal.go:HasParentFilePath:464 → stat .git: no such file or directory
[DEBUG] 10:55:23.438 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.438 terminal.go:HasParentFilePath(.git) - 28.899µs
[TRACE] 10:55:23.438 store.go:](session toggle_cache) - 9.024µs
[TRACE] 10:55:23.438 terminal.go:Flags() - 42ns
[DEBUG] 10:55:23.438 segment.go:isToggled:291 → no toggles found
[TRACE] 10:55:23.438 terminal.go:GOOS() - 68ns
[DEBUG] 10:55:23.438 store.go:]:176 → (device) found entry: environment_platform - ubuntu
[TRACE] 10:55:23.438 store.go:](device environment_platform) - 4.525µs
[DEBUG] 10:55:23.438 map.go:Bool:133 → display_distro_name: false
[TRACE] 10:55:23.438 terminal.go:Flags() - 257ns
[DEBUG] 10:55:23.438 map.go:String:79 → ubuntu: 
[DEBUG] 10:55:23.438 home.go:12 → /home/anwer
[TRACE] 10:55:23.438 terminal.go:GOOS() - 85ns
[DEBUG] 10:55:23.438 segment.go:Execute:173 → segment: Path
[TRACE] 10:55:23.438 terminal.go:Flags() - 246ns
[DEBUG] 10:55:23.438 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.438 store.go:](session toggle_cache) - 9.562µs
[DEBUG] 10:55:23.438 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.438 map.go:Bool:133 → display_cygpath: false
[TRACE] 10:55:23.438 terminal.go:GOOS() - 26ns
[TRACE] 10:55:23.438 separator.go:Separator() - 60ns
[TRACE] 10:55:23.438 terminal.go:Shell() - 109ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 27ns
[DEBUG] 10:55:23.438 map.go:Bool:133 → mapped_locations_enabled: true
[DEBUG] 10:55:23.438 map.go:String:79 → windows_registry_icon: 
[DEBUG] 10:55:23.438 map.go:String:79 → windows_registry_icon: 
[DEBUG] 10:55:23.438 home.go:12 → /home/anwer
[TRACE] 10:55:23.438 separator.go:Separator() - 41ns
[DEBUG] 10:55:23.438 text.go:patchTemplate:149 →  {{ if .WSL }}WSL at {{ end }}{{.Data.Icon}}
[TRACE] 10:55:23.438 terminal.go:GOOS() - 56ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 25ns
[DEBUG] 10:55:23.438 map.go:String:83 → home_icon: ~
[DEBUG] 10:55:23.438 map.go:KeyValueMap:208 → mapped_locations: map[]
[TRACE] 10:55:23.438 separator.go:Separator() - 43ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 42ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 43ns
[TRACE] 10:55:23.438 separator.go:Separator() - 28ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 25ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 32ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 26ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 26ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 24ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 82ns
[DEBUG] 10:55:23.438 map.go:StringArray:220 → cycle: []
[DEBUG] 10:55:23.438 map.go:String:79 → folder_separator_template:
[DEBUG] 10:55:23.438 map.go:String:79 → folder_separator_icon: /
[DEBUG] 10:55:23.438 map.go:Bool:133 → cycle_folder_separator: false
[DEBUG] 10:55:23.438 map.go:String:79 → folder_format: %s
[DEBUG] 10:55:23.438 map.go:String:79 → edge_format: %s
[DEBUG] 10:55:23.438 map.go:String:79 → left_format: %s
[DEBUG] 10:55:23.438 map.go:String:79 → right_format: %s
[TRACE] 10:55:23.438 terminal.go:Flags() - 27ns
[TRACE] 10:55:23.438 terminal.go:GOOS() - 26ns
[TRACE] 10:55:23.438 terminal.go:StackCount() - 52ns
[TRACE] 10:55:23.438 terminal_unix.go:DirIsWritable(/home/anwer) - 6.357µs
[TRACE] 10:55:23.438 terminal.go:Flags() - 29ns
[DEBUG] 10:55:23.438 text.go:patchTemplate:149 →   {{ .Data.Path }}
[TRACE] 10:55:23.438 terminal.go:Shell() - 270ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 42ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 106ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 33ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 26ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 89ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 45ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 27ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 26ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 26ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 25ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 23ns
[TRACE] 10:55:23.438 terminal.go:Flags() - 42ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 54ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Time
[TRACE] 10:55:23.439 terminal.go:Flags() - 249ns
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 5.319µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 map.go:String:79 → time_format: 15:04:05
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 84ns
[TRACE] 10:55:23.439 terminal.go:Flags() - 60ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Julia
[TRACE] 10:55:23.439 terminal.go:Flags() - 23ns
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 5.392µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 26ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Python
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 3.913µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 map.go:String:83 → display_mode: files
[DEBUG] 10:55:23.439 map.go:StringArray:220 → extensions: [*.jl]
[DEBUG] 10:55:23.439 map.go:StringArray:220 → folders: []
[DEBUG] 10:55:23.439 map.go:Bool:133 → home_enabled: false
[TRACE] 10:55:23.439 terminal.go:Flags() - 240ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:Flags() - 88ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:Flags() - 49ns
[TRACE] 10:55:23.439 terminal.go:GOOS() - 73ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Node
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 94ns
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Go
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 9.027µs
[DEBUG] 10:55:23.439 map.go:StringArray:220 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 6.508µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 map.go:StringArray:220 → folders: [.venv venv virtualenv venv-win pyenv-win]
[TRACE] 10:55:23.439 terminal.go:Flags() - 240ns
[TRACE] 10:55:23.439 terminal.go:Flags() - 143ns
[DEBUG] 10:55:23.439 map.go:StringArray:220 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 122ns
[DEBUG] 10:55:23.439 map.go:StringArray:220 → folders: []
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Root
[DEBUG] 10:55:23.439 map.go:Bool:133 → home_enabled: false
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:Flags() - 64ns
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 terminal.go:Flags() - 248ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 70ns
[DEBUG] 10:55:23.439 map.go:StringArray:220 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 11.021µs
[DEBUG] 10:55:23.439 map.go:StringArray:220 → folders: []
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 map.go:Bool:133 → home_enabled: false
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:Flags() - 59ns
[TRACE] 10:55:23.439 terminal_unix.go:Root() - 952ns
[TRACE] 10:55:23.439 terminal.go:Flags() - 72ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Aws
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 16.562µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.439 map.go:Bool:141 → display_default: false
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_VAULT) - 12.19µs
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_DEFAULT_PROFILE) - 5.84µs
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_PROFILE) - 2.897µs
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_REGION) - 4.224µs
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_DEFAULT_REGION) - 4.785µs
[DEBUG] 10:55:23.439 terminal.go:Getenv:64 → NO DATA
[TRACE] 10:55:23.439 terminal.go:Getenv(AWS_CONFIG_FILE) - 2.627µs
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[ERROR] 10:55:23.439 terminal.go:FileContent:207 → open /home/anwer/.aws/config: no such file or directory
[TRACE] 10:55:23.439 terminal.go:FileContent(/home/anwer/.aws/config) - 15.189µs
[TRACE] 10:55:23.439 terminal.go:Flags() - 49ns
[TRACE] 10:55:23.439 terminal.go:Flags() - 240ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[TRACE] 10:55:23.439 terminal.go:GOOS() - 137ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Ruby
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.439 store.go:](session toggle_cache) - 15.443µs
[DEBUG] 10:55:23.439 segment.go:isToggled:291 → no toggles found
[TRACE] 10:55:23.439 terminal.go:GOOS() - 142ns
[TRACE] 10:55:23.439 terminal.go:Flags() - 43ns
[DEBUG] 10:55:23.439 home.go:12 → /home/anwer
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Exit
[TRACE] 10:55:23.439 terminal.go:GOOS() - 104ns
[DEBUG] 10:55:23.439 segment.go:Execute:173 → segment: Executiontime
[DEBUG] 10:55:23.440 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.440 store.go:](session toggle_cache) - 7.649µs
[DEBUG] 10:55:23.440 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.440 map.go:Bool:133 → always_enabled: false
[TRACE] 10:55:23.440 terminal.go:ExecutionTime() - 68ns
[DEBUG] 10:55:23.440 map.go:Float64:148 → threshold: 500.000000
[TRACE] 10:55:23.440 terminal.go:Flags() - 54ns
[TRACE] 10:55:23.440 terminal.go:Flags() - 129ns
[DEBUG] 10:55:23.439 store.go:]:176 → (session) found entry: toggle_cache - map[]
[DEBUG] 10:55:23.440 home.go:12 → /home/anwer
[TRACE] 10:55:23.440 terminal.go:GOOS() - 41ns
[DEBUG] 10:55:23.440 segment.go:Execute:173 → segment: Azfunc
[DEBUG] 10:55:23.440 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.440 store.go:](session toggle_cache) - 3.916µs
[DEBUG] 10:55:23.440 segment.go:isToggled:291 → no toggles found
[TRACE] 10:55:23.440 store.go:](session toggle_cache) - 52.219µs
[DEBUG] 10:55:23.440 map.go:StringArray:220 → folders: []
[DEBUG] 10:55:23.440 segment.go:isToggled:291 → no toggles found
[DEBUG] 10:55:23.440 map.go:Bool:133 → home_enabled: false
[DEBUG] 10:55:23.440 home.go:12 → /home/anwer
[TRACE] 10:55:23.440 terminal.go:Flags() - 25ns
[TRACE] 10:55:23.440 terminal.go:StatusCodes() - 216ns
[DEBUG] 10:55:23.440 map.go:String:79 → status_template: {{ .Code }}
[DEBUG] 10:55:23.440 map.go:Bool:133 → home_enabled: false
[DEBUG] 10:55:23.440 home.go:12 → /home/anwer
[TRACE] 10:55:23.440 terminal.go:Flags() - 27ns
[DEBUG] 10:55:23.440 map.go:StringArray:220 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 10:55:23.440 map.go:StringArray:220 → folders: []
[DEBUG] 10:55:23.440 map.go:Bool:133 → home_enabled: false
[DEBUG] 10:55:23.440 home.go:12 → /home/anwer
[TRACE] 10:55:23.440 terminal.go:Flags() - 48ns
[DEBUG] 10:55:23.440 text.go:patchTemplate:149 → {{ .Code }}
[DEBUG] 10:55:23.440 map.go:Bool:141 → always_enabled: true
[TRACE] 10:55:23.440 terminal.go:Flags() - 251ns
[DEBUG] 10:55:23.440 text.go:patchTemplate:149 →  {{ if gt .Code 0 }} {{ .Data.Meaning }}{{ else }}✔{{ end }}
[DEBUG] 10:55:23.441 text.go:patchTemplate:149 → {{ if gt .Code 0 }}#cc2222{{ end }}
[DEBUG] 10:55:23.441 text.go:patchTemplate:149 → {{ if gt .Code 0 }}#cc2222{{ end }}
[DEBUG] 10:55:23.441 text.go:patchTemplate:149 →  {{ .Data.CurrentDate | date .Data.Format }} 
[DEBUG] 10:55:23.441 text.go:patchTemplate:149 → {{ if gt .Code 0 }}#cc2222{{ end }}
[DEBUG] 10:55:23.444 terminal_unix.go:TerminalWidth:83 → terminal width: 125
[TRACE] 10:55:23.444 terminal_unix.go:TerminalWidth() - 3.081828ms
[DEBUG] 10:55:23.444 engine.go:shouldFill:165 → no filler specified
[TRACE] 10:55:23.444 terminal.go:Shell() - 165ns
[TRACE] 10:55:23.444 terminal.go:Flags() - 40ns
[TRACE] 10:55:23.444 terminal.go:Flags() - 24ns
[DEBUG] 10:55:23.444 home.go:12 → /home/anwer
[TRACE] 10:55:23.444 terminal.go:GOOS() - 62ns
[DEBUG] 10:55:23.444 segment.go:Execute:173 → segment: Text
[DEBUG] 10:55:23.444 store.go:]:176 → (session) found entry: toggle_cache - map[]
[TRACE] 10:55:23.444 store.go:](session toggle_cache) - 14.56µs
[DEBUG] 10:55:23.444 segment.go:isToggled:291 → no toggles found
[TRACE] 10:55:23.444 terminal.go:Flags() - 104ns
[TRACE] 10:55:23.444 terminal.go:Shell() - 386ns
[TRACE] 10:55:23.444 terminal.go:Flags() - 64ns
[TRACE] 10:55:23.444 terminal.go:Shell() - 211ns
[TRACE] 10:55:23.444 path.go:Path() - 285ns
```

--- a/Casks/oh-my-posh.rb
+++ b/Casks/oh-my-posh.rb
@@ -1,14 +1,17 @@
 cask "oh-my-posh" do
     desc "Prompt theme engine for any shell"
     homepage "https://ohmyposh.dev"
-    arch arm: "arm64", intel: "amd64"
     version "29.19.0"
-    url "https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v#{version}/posh-darwin-#{arch}",
-        verified: "github.com/JanDeDobbeleer/oh-my-posh/"
-    sha256 arm:   "23cb0817c60296ab64ec8fc4be43ffdfca7405ef9e0b1e076149ba5413fb08ac",
-           intel: "d94b878ac059f16acbb9aff2056b3d37c6d513a8eca1e803aaaaeb4d202c0036"
     name "oh-my-posh"
-    binary "posh-darwin-#{arch}", target: "oh-my-posh"
+
+    on_macos do
+        arch arm: "arm64", intel: "amd64"
+        url "https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v#{version}/posh-darwin-#{arch}",
+            verified: "github.com/JanDeDobbeleer/oh-my-posh/"
+        sha256 arm:   "23cb0817c60296ab64ec8fc4be43ffdfca7405ef9e0b1e076149ba5413fb08ac",
+               intel: "d94b878ac059f16acbb9aff2056b3d37c6d513a8eca1e803aaaaeb4d202c0036"
+        binary "posh-darwin-#{arch}", target: "oh-my-posh"
+    end
+
     auto_updates true
 end
-

--- a/build/cask.rb
+++ b/build/cask.rb
@@ -1,13 +1,17 @@
 cask "oh-my-posh" do
     desc "Prompt theme engine for any shell"
     homepage "https://ohmyposh.dev"
-    arch arm: "arm64", intel: "amd64"
     version "<VERSION>"
-    url "https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v#{version}/posh-darwin-#{arch}",
-        verified: "github.com/JanDeDobbeleer/oh-my-posh/"
-    sha256 arm:   "<SHA256-ARM>",
-           intel: "<SHA256-INTEL>"
     name "oh-my-posh"
-    binary "posh-darwin-#{arch}", target: "oh-my-posh"
+
+    on_macos do
+        arch arm: "arm64", intel: "amd64"
+        url "https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v#{version}/posh-darwin-#{arch}",
+            verified: "github.com/JanDeDobbeleer/oh-my-posh/"
+        sha256 arm:   "<SHA256-ARM>",
+               intel: "<SHA256-INTEL>"
+        binary "posh-darwin-#{arch}", target: "oh-my-posh"
+    end
+
     auto_updates true
 end

